defaults/main.yml

```diff
@@ -39,3 +41,9 @@ cartridge_scenario:
   - setup_app_config
   - bootstrap_vshard
   - setup_failover
+
+cartridge_create_systemd_unit_files: true
+cartridge_systemd_dir: /etc/systemd/system
```

Comment:
Is't necessary defaults values for cartridge_run_dir, cartridge_data_dir, cartridge_conf_dir, cartridge_dist_dir?

Reply:
Yes, of course. These values are used for rpm and deb.